### ISSUE TITLE
[Package Request]: Microsoft.XboxInstaller (or Microsoft.Xbox)

### ISSUE THREAD

**Issue by DeusMaximus** [microsoft/winget-pkgs#51031]
### Package Requested

- [ ] I would like someone else to build the manifest.
- [X] I would like help so I can submit the manifest.
- [X] I have performed a search and couldn't find this package.
- [X] I have checked that the installer for this package is not a standalone or portable executable or a .zip file
- [ ] I think there is a new version available and I have provided the URL.

### Please provide the following information

Publisher: Microsoft
Publisher Url: https://www.xbox.com/en-US/apps/xbox-app-for-pc
Package Name: XboxInstaller
Description: Discover and download new games with Game Pass, play console games on your Windows PC with cloud gaming, and chat with friends across PC, mobile, and Xbox console.
Package Version: 1.0.1.14 (App store app is 2202.1001.9.0 as of time of writing)
Installer URL: https://assets.xbox.com/installer/20190628.8/anycpu/XboxInstaller.exe (from https://www.xbox.com/en-US/apps/xbox-app-for-pc)

Basically, XboxInstaller.exe is a wrapper for installing the Xbox app from the store. One potential issue I can see with creating a manifest for this is the fact that no traditional uninstall string is created in the registry as it's just a wrapper for installing the store app, so uninstalling or checking the version number in winget would probably be impossible.

Also, I'm not sure if it supports being installed silently, either.

**Comment by OfficialEsco** [comment]
I doooon't think its a smart idea to add this since its like you said, its purely a executable "installer".

When trying to use the "Installer" in Sandbox i got a 0x80070422 error code after pressing Install which pointed to a Windows Update issue, when enabling the Windows Update service it installed successfully.

After the installation the installed Entries are:
```
Publisher: Microsoft Corporation
DisplayName: Xbox
DisplayVersion: 2202.1001.9.0
InstallerType: msstore
PackageFamilyName: Microsoft.GamingApp_2202.1001.9.0_x64__8wekyb3d8bbwe
```
```
Publisher: Microsoft Corporation
DisplayName: Xbox Identity Provider
DisplayVersion: 12.85.31001.0
InstallerType: msstore
PackageFamilyName: Microsoft.XboxIdentityProvider_12.85.31001.0_x64__8wekyb3d8bbwe
```
```
Publisher: Microsoft Corporation
DisplayName: Gaming Services
DisplayVersion: 3.62.18001.0
InstallerType: msstore
PackageFamilyName: Microsoft.GamingServices_3.62.18001.0_x64__8wekyb3d8bbwe
```

This means it cannot be updated and would most likely cause a conflict with the msstore package.

As of today you can search, install and show the Xbox app via WinGet as long as you have the msstore source with
* 9MV0B5HZVK9Z
* Xbox

Hopefully in the future the msstore implementation will provide a better experience.

**Comment by DandelionSprout** [comment]
In the years that have passed since the issue report was created, the `assets.xbox.com` domain itself seems to have become dead, which unfortunately is certainly also an obstacle for this package.
